pages/apis/rest_api/team_pipelines.md

```diff
@@ -0,0 +1,174 @@
+# Team pipelines API
+
+The team pipelines API allows users to review, create, update and destroy pipelines associated with a team in your organization.
+
+## Team pipeline data model
+
+<table class="responsive-table">
+<tbody>
+  <tr><th><code>pipeline_id</code></th><td>UUID of the associated pipeline</td></tr>
+  <tr><th><code>access_level</code></th><td>The access levels that users have to the associated pipeline - <code>read_only</code>, <code>build_and_read</code>, <code>manage_build_and_read</code></td></tr>
```

Comment:
This reads as a user being able to have multiple access levels. Is that right?

Reply:
I can change to 'level' 👍